### ISSUE TITLE
Static files not being copied to _book output dir with 3.0.0-pre.12

### ISSUE THREAD

**Issue by inetfuture** [GitbookIO/gitbook#1272]
file structure:

```
assets/
    jquery.js
SUMMARY.md
...
```

`gitbook build`:

```
info: 7 plugins are installed 
info: loading plugin "highlight"... OK 
info: loading plugin "search"... OK 
info: loading plugin "lunr"... OK 
info: loading plugin "sharing"... OK 
info: loading plugin "fontsettings"... OK 
info: loading plugin "theme-default"... OK 
info: found 4 pages 
info: found 0 asset files 
info: >> generation finished with success in 2.7s ! 
```

**Comment by inetfuture** [comment]
Downgrade to `2.6.7` fixes the issue 😭

**Comment by SamyPesse** [comment]
Fixed in `3.0.0-pre.13`
